dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -82,6 +82,16 @@ def replace_hostname(url)
   @browser.execute_script("$(\"#{jquery_selector}\").click();")
 end
 
+When /^I press delete$/ do
+  script = "Blockly.mainBlockSpaceEditor.onKeyDown_("
+  script +="{"
+  script +="  target: {},"
+  script +="  preventDefault: function() {},"
+  script +="  keyCode: $.simulate.keyCode['DELETE']"
+  script +="})"
+  @browser.execute_script(script)
+end
```

Comment:
I think we've maybe had issues like that before. Your approach seems okay. I guess what it would miss is the case where delete doesnt get translated to keydown for some reason.